modules/eventing/pages/eventing-language-constructs.adoc

```diff
@@ -392,35 +392,21 @@ Integrating Eventing with Analytics:
 [source,javascript]
 ----
 function OnUpdate(doc, meta) {
-    // Ignore information we don't care about
-    if (doc.type !== 'airline') return;
-
-    // Get the total routes per IATA
-    var route_cnt = 0;     
-    // Uses a true variable as a SQL++ parameter  
-    var airline = doc.iata;  
-
-    var results = ANALYTICS(
-        "SELECT COUNT(*) AS cnt
-        FROM `travel-sample`.`inventory`.`route`
-        WHERE type = \"route\" 
-		AND airline = $1", [doc.iata]
-    );
-
-    // Stream results using the 'for' iterator
-    for (var item of results) {   
-        route_cnt = item.cnt;
-    }
+  var count = 0;
+  const limit = 4;
 
-    // End the query and free the resources held
-    results.close();    
+  let query = couchbase.analyticsQuery('SELECT * FROM default LIMIT $limit;', {"limit": limit});
```

Comment:
A precondition for this analytics query to successfully execute is that the analytics dataset called `default` must exist.

It would be helpful to either add a comment in the program or a note point around it mentioning this.

Reply:
Done